iothub/device/src/DeviceClient.cs

```diff
@@ -527,8 +527,9 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
             InternalClient.CompleteFileUploadAsync(notification, cancellationToken);
 
         /// <summary>
-        /// Registers a new delegate for the named method. If a delegate is already associated with
+        /// Sets a new delegate for the named method. If a delegate is already associated with
```

Comment:
Oh, you are updating the text here to say "Sets" instead of "Registers"! Now I understand your comment on the ReceiveAsync() PR! 😉 

Do you prefer "set and unset" vs "register and deregister"?

Reply:
David thinks that's much simpler and easy to understand :)

Reply:
Ok then, would you mind scanning through the rest of the comments as well, and make this change.

Reply:
I had done it for all but may be the newer one may not be following this, is that what you meant?

Reply:
Changed it for new callback you added